lib/modules/home/components/top_bar.dart

```diff
@@ -20,12 +20,10 @@ class TopBar extends StatelessWidget {
                   Bone(
                     height: 18,
                     width: 117,
-                    child: StreamsSelector0<DateTime>.value(
-                      stream: state._dateTime,
-                      builder: (_, dateTime, __) => Text(
-                        DateFormat("EEEE, MMM d")
-                            .format(dateTime)
-                            .toUpperCase(),
+                    child: StreamsSelector0<String>.value(
+                      stream: state._dateTimeText,
+                      builder: (_, text, __) => Text(
```

Comment:
minor, but I prefer the previous way you named the argument similar to the observed stream more than this ambiguous name `text`. `_dateTimeText` then -> `builder: (_, dateTimeText, _`)`